packages/notifi-dataplane/lib/types/FusionMessage.ts

```diff
@@ -1,33 +1,12 @@
 import { Types } from 'notifi-graphql/dist';
 
-// If payload type undefined, fallback to default CommunityManagerJsonPayload
-export interface FusionMessage<T extends object = CommunityManagerJsonPayload> {
+export interface FusionMessage {
```

Comment:
```suggestion
export interface FusionMessage<T extends object = object> {
```

Reply:
Here is an example after change:
```ts
const vars: Readonly<FusionMessage<{ x: number; y: number }>[]> = [
    {
      eventTypeId: 'eventTypeId',
      variablesJson: { x: 1, y: 2 },
    },
  ];


  return client
    .publishFusionMessage(jwt, vars)
```

<img width="593" alt="Screenshot 2024-02-16 at 10 07 08" src="https://github.com/notifi-network/notifi-sdk-ts/assets/127958634/2ebfe561-4aa8-403f-a1a1-ecde8f7e1ed6">